Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/projects/SparkVersion.java

```diff
@@ -25,6 +25,7 @@
 import java.util.Comparator;
 
 public enum SparkVersion {
+    SPARK_2_2_0("2.2.0", "2.11.8", "2.11"),

```

Comment:
>SPARK_2_2_0("2.2.0", "2.11.8", "2.11"), [](start = 4, length = 39)

Please do test all of our samples against Spark 2.2 cluster.

Reply:
Ok, I will finish all the sample tests before I merge this PR.